test/wkok/openai_clojure_test.clj

```diff
@@ -16,3 +16,22 @@
                                            :max_tokens 7
                                            :temperature 0})
                    :choices))))
+
+
+(comment
+  (api/create-completion {:model "text-davinci-003"
+                          :prompt "Say this is a test"
+                          :max_tokens 7
+                          :temperature 0})
+
+  (api/create-completion :openai {:model "text-davinci-003"
+                                  :prompt "Say this is a test"
+                                  :max_tokens 7
+                                  :temperature 0})
+
+
+  (api/create-completion :azure
+                         {:martian.core/body {:prompt "How are you ?"}
```

Comment:
The :body of :azure takes exactly the same params  (except :model) then :openai
Not sure, I want to harmonize this as well

Reply:
The logic for :azure using flat params could be
"put all in :martian/body, except :deployment-id and :api-version"

Reply:
I implemented the further harmonisation.
So now openai and Azure OpenAI take the same input map and I patch it before sending to :azure
(for completion at least)
The only remaining difference is the additional (required) :api-version, which I need to think about how to handle

Reply:
The openai API is explicitely versioned as well. (currently "v1")
So probably the code should gain "generic" handling of API versions, which either
adds to path "v1" (openai)
or adds url parameter (?api-version=)

Reply:
I hard coded the azure API version for now.
(as the :openai version is as well implicitly hardcoded currently to "v1")
We can address this in an other change.